clouddriver-api/src/main/java/com/netflix/spinnaker/clouddriver/deploy/ExtensibleDescriptionValidator.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2021 Armory, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.clouddriver.deploy;
+
+import com.netflix.spinnaker.kork.annotations.Beta;
+import java.util.List;
+
+@Beta
+public interface ExtensibleDescriptionValidator {
```

Comment:
I would make this extend `SpinnakerExtensionPoint`.

Reply:
I would consider changing the name. Most description validators work on a single description type. This kind of description validator works on all description types. You could call it `GlobalDescriptionValidator`. 

You could also consider whether it should be flexible and have another method like `boolean handles(...)` that determines if it should handle the description type. For your use-case, `handles` would just always return `true`.

Reply:
Could you please also add docs comments to the interface and the method?

Reply:
Sounds good, adding these changes 👍